cura/Settings/GlobalStack.py

```diff
@@ -301,6 +301,16 @@ def isValid(self) -> bool:
                 return False
         return True
 
+    def isInOneAtATimeMode(self) -> bool:
+        """
+        Gets whether the one-at-a-time mode is activated, provided that there is only one active extruder
+        """
+        one_at_a_time_mode_is_active = self.getProperty("print_sequence", "value") == "one_at_a_time"
+        extruders_enabled_count = self.definitionChanges.getProperty("extruders_enabled_count", "value")
```

Comment:
Isn't it just easier to look at the enabled property? Now you are effectively re-implementing the enabled check.

Reply:
Indeed, I just figured that it is clearer using the extruders enable count